snowplow/iglu-client-embedded/schemas/com.metabase/event/jsonschema/1-0-0

```diff
@@ -0,0 +1,63 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Generic event to track interactions and actions that happen within Metabase. This should be used as a default schema for tracking events. If more specific context is needed, we can use custom context entities (https://docs.snowplow.io/docs/understanding-your-pipeline/entities/).",
+    "self": {
+      "vendor": "com.metabase",
+      "name": "event",
+      "format": "jsonschema",
+      "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+      "event": {
+        "description": "Name of the action. Noun (target) + Verb in the past (action) which define the action taken - e.g. question_created, dashboard_updated, dashboard_auto_apply_filter_enabled",
+        "type": "string",
+        "maxLength": 1024
+      },
+      "target_id": {
```

Comment:
@Somtom last minute thought. In the app all ids are integers. Should we make it an integer?

Reply:
Pushed the change for now but please take a look